cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -76,6 +76,14 @@ func RedefinePodWithNetwork(pod *corev1.Pod, networksSpec string) *corev1.Pod {
 	return pod
 }
 
+// RedefineAsPrivileged updates the pod definition to be privileged
+func RedefineAsPrivileged(pod *corev1.Pod) *corev1.Pod {
+	pod.Spec.Containers[0].SecurityContext = &corev1.SecurityContext{}
+	b := true
+	pod.Spec.Containers[0].SecurityContext.Privileged = &b
```

Comment:
doesn't it make only the first container privileged, instead of the whole pod?

Reply:
It does. Like some other Redefine/Define functions in the file it changes only the first container so I went with it. Also from what I see there isn't a `privileged` parameter for a Pod, and I'm not sure updating all of the pod's containers is necessary here. btw I took it from here https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/test/util/pod/pod.go#L56